source/game/field/obj/ObjectColMgr.hh

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "game/field/CourseColMgr.hh"
+#include "game/field/KColData.hh"
+
+#include <egg/math/BoundBox.hh>
+#include <egg/math/Matrix.hh>
+
+namespace Field {
+
+class ObjectColMgr {
+public:
+    static void *LoadFile(const char *filename);
```

Comment:
`static` members come after non-static members. Move to end of `public` section. Also be sure to move the definition in the source accordingly.